source/install/prod-docker.rst

```diff
@@ -51,4 +51,11 @@ Production Docker Setup on Mac OS X
 
 You can run a production deployment on Mac OS X by `installing Docker Compose using the online guide <http://docs.docker.com/installation/mac/>`_ then following the above instructions.
 
-**Other options:** To install a feature-equivalent version of Mattermost that does not upgrade to enterprise features using a license key, Mattermost Team Edition, repeat steps above excluding ``-b enterprise`` from ``git clone`` command.
+Other options:
+--------------
+
+To install a feature-equivalent version of Mattermost that does not upgrade to enterprise features using a license key, Mattermost Team Edition, open ``docker-compose.yaml`` and comment out the following line:
```

Comment:
I took out the "feature equivalent" stuff. It's superfluous, and we don't have that type of info in any of the other install docs.